### DISCUSSION TITLE
'C' Disc no space available

### DISCUSSION THREAD

**Discussion by akmalwest** [Chia-Network/chia-blockchain#11323] [Support]
160 GB of 232 GB available is taken by Chia in this folder, pfa: 
![image](https://user-images.githubusercontent.com/82469293/165390543-6c7c2f5c-0984-4426-9992-105bd015d637.png)

Is there a way i can free up space in the C drive? 

Please advise.

**Comment by emlowe** [comment]
https://thechiaplot.net/2022/02/12/how-to-move-your-chia-database-to-a-new-disk/

**Comment by Kaecki** [comment]
Hallo,
leider funktioniert das bei mir nicht wie dort beschrieben. Bei mir steht immer nur "Das System kann den angegebenen Pfad nicht finden". Was mache ich falsch???

**Comment by Scrooghy** [comment]
Blockchain transfer from drive C to drive D

in cmd

mklink /j C:\Users\User\ .chia D:\ .chia

remove spaces before .chia

**Reply by akmalwest** [reply]
Can you please share the exact code to do this?

**Comment by Scrooghy** [comment]
mklink /j C:\Users\User\.chia D:\.chia

User - your username
D:\.chia - your new directory

пн, 2 мая 2022 г. в 15:33, akmalwest ***@***.***>:

**Comment by Scrooghy** [comment]
1. Shotdown node Chia and check, kill all processes.
2. Copy directory .chia from C:\Users\User\ to new place, example D:\
3. Del .chia from C:\Users\User\
4. Run CMD as Administrator. NOT PowerShell! Put command: mklink /j C:\Users\User\ .chia D:\ .chia
(remove spaces before .chia) Enter.
5. End :)

**Reply by akmalwest** [reply]
Sorry,

1. Done
2. Instead of copying, i cut the folder from C and pasted in D
3. There was no need for that for the reason in 2
4. I am getting the error as in the attachment 

Please advise..
![Capture](https://user-images.githubusercontent.com/82469293/169024616-8497270b-7453-44b3-aed3-705c70789b3f.PNG)

**Reply by akmalwest** [reply]
Sorry for being dumb, some people are not handy at these stuff..

So, why am I seeing the error as in the attachment? 

Please tell me how to fix it 

Once again, I am sorry

**Comment by Scrooghy** [comment]
I wrote everything

ср, 18 мая 2022 г. в 15:05, akmalwest ***@***.***>:
